src/tests/SimTest.cpp

```diff
@@ -797,6 +799,61 @@ void SimTest::addSensorToObject() {
       Cr::Utility::Directory::join(screenshotDir, "SimTestExpectedScene.png"),
       (Mn::DebugTools::CompareImageToFile{maxThreshold, 0.75f}));
 }
+
+void SimTest::createMagnumRenderingOff() {
+  Corrade::Utility::Debug() << "Starting Test : createMagnumRenderingOff ";
+
+  SimulatorConfiguration simConfig{};
+  simConfig.activeSceneName = planeStage;
+  simConfig.enablePhysics = true;
+  simConfig.physicsConfigFile = physicsConfigFile;
+  simConfig.overrideSceneLightDefaults = true;
+  simConfig.createRenderer = false;
+  simConfig.sceneLightSetup = esp::NO_LIGHT_KEY;
+
+  // check that creating a simulator and adding a primitive object works
+  auto simulator = Simulator::create_unique(simConfig);
+  auto objectAttribsMgr = simulator->getObjectAttributesManager();
+  int objectID = simulator->addObjectByHandle("cubeSolid");
```

Comment:
Can you confirm whether adding this object tests the PbrDrawable codepath? I guess you could verify this via the VS code debugger or by adding a print statement locally.

Reply:
This does not; currently it seems like making `createMagnumRenderer=true` is the difference between `PbrDrawable` being tested or not, which defeats the purpose. I'll keep looking to see if there is a way for the codepath to be executed when the flag is set to `false`. If it turns out not to be possible, I'll remove the changes in `PbrDrawable.cpp`.

Reply:
Yep, I did some backtracking and turns out it's impossible to trigger the PbrDrawable codepath when `createRenderer=false`.

This is because in `ResourceManager.cpp`, the only time a Pbr material can possibly be added to `shaderManager_` is for `finalMaterial` in `loadMaterials`.

However, `loadMaterials` cannot possibly run because it requires `requiresTextures_=true`, but in `Simulator.cpp` we have
```
if (!config_.createRenderer) {
  config_.requiresTextures = false;
}
```